code/index.html

```diff
@@ -1,22 +1,116 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta cnarset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <!-- dont forget the meta tag for responsiveness -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="icon" href="./img/favicon-2.png" type="image/x-icon">
+  <title>Japan Travel Magazine</title>
+</head>
+
+<body>
+  <header>
+    <div class="container clear">
+      <div class="header-left">
+        <h1 class="header-nav-logo">
+          <a href="#"><img src="./img/logo.png" alt="">
+            <h1>Japan Travel Magazine</h1>
+        </h1>
+      </div>
+      <div class="header-right">
+        <ul class="header-nav-clear">
+          <li class="header-nav-item "><a href="#">News</a></li>
+          <li class="header-nav-item "><a href="#">About</a></li>
+          <li class="header-nav-item "><a href="#">More</a></li>
+          <li class="header-nav-item "><a href="#">Share</a></li>
+        </ul>
+      </div>
+    </div>
+  </header>
+
+  <section class="big-news section">
+    <!-- The big news content -->
+    <div class="container clear">
+      <div class="about-left">
+        <img src="./img/top-jtm.jpg" alt="mt.fuji">
+      </div>
+      <div class="about-right">
+        <p> Visit Japan.orem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec tempus purus. Fusce nec dui
+          rutrum, iaculis lorem sagittis, porttitor purus. Mauris interdum, nulla ut faucibus tristique, ligula quam
+          faucibus odio, a vulputate massa tellus in justo. Donec eget est bibendum, fermentum quam vitae, imperdiet
+          purus. Curabitur pellentesque placerat ante vel ultrices. Maecenas hendrerit accumsan lacinia. Nam posuere
+          felis mi, in pharetra augue vestibulum sed. Praesent et massa efficitur metus laoreet laoreet ac eget ligula.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="news-category">
+    <div class="conteiner clear">
+      <h2 class="section-title">News</h2>
+      <div class="news-wrap">
+        <div class="contents-box">
+          <img src="./img/news-1.jpg" alt="flexbox">
```

Comment:
Also in the alt attributes for the card images I think we shall have a description of the image instead of "flexbox"